src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -243,19 +622,52 @@
 		"sourcePath" : "log.sagebase.org/fileUploadRecords/",
 		"tableDescriptor": {
 			"name": "fileuploadrecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"user_id": "bigint",
-				"project_id": "bigint",
-				"file_handle_id": "bigint",
-				"association_object_id": "bigint",
-				"association_object_type": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"record_date": "date"
+			"description" : "The table contains file upload records. Records contains information regarding upload of a file on Synapse.",
+			"columns":[{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The timestamp of the file upload."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The identifier of user who uploaded the file."
 			},
+			{
+				"name": "project_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the project where the file resides."
```

Comment:
What is the file is not a file entity?  E.g., if the file is a wiki attachment, user profile picture, or team icon, what values goes here?